### PR TITLE
[RFR] Gemify

### PR DESCRIPTION
**Pull request by KittyGiraudel**
Ping @cadejscroggins for review. :)

### CODE REVIEW

File: spritesh.gemspec [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require 'json'
+package = JSON.parse(IO.read("package.json"))
+
+Gem::Specification.new do |spec|
+  spec.name          = package["name"]
```
Comment (nburkley) [inline spritesh.gemspec:5]: These should be filled out

**Comment by nburkley** [comment]
Looks fine, not much to comment on as it's just a wrapper around the script. The gemspec should be filled out name, version, description, homepage and license

**Comment by KittyGiraudel** [comment]
They are. We just read it from the package.json file.

**Comment by nburkley** [comment]
Ah my bad. This is what happens when a JS dev writes a ruby gem.

**Comment by KittyGiraudel** [comment]
Hahaha! :D

**Comment by xvvvyz** [comment]
Looks good. As long as you're cool with having different names for the executable (sprite.sh for npm and spritesh for gem).

**Comment by KittyGiraudel** [comment]
Actually the npm executable won’t be `sprite.sh` but `spritesh`. In package.json, `"bin": "bin/sprite.sh"` is a shortcut for `"bin": { "spritesh": "bin/sprite.sh" }`. The executable is `spritesh`.

**Comment by xvvvyz** [comment]
Ahh. In that case. Looks great! We just won't be able to push to RubyGems until version 1.0.8 thanks to me (:

**Comment by xvvvyz** [comment]
I just noticed though—the Ruby wrapper needs to have its permissions set to 755.

**Comment by KittyGiraudel** [comment]
Done! Nice catch. :)

**Comment by KittyGiraudel** [comment]
Thank you very much for your time @cadejscroggins, super happy to have everything here. :)

**Comment by xvvvyz** [comment]
Not a problem. Glad I could help. Thanks for making it in the first place! Has saved me a bit of hassle.

**Pull request merged**